packages/headless/src/auth/middleware.ts

```diff
@@ -3,6 +3,16 @@ import { authorize } from './authorize';
 import { storeAccessToken } from './cookie';
 import { getQueryParam } from '../utils';
 
+/**
+ * A Node handler for processing incomming requests to exchange an Authorization Code
```

Comment:
Tiny typo in 'incomming':

```suggestion
 * A Node handler for processing incoming requests to exchange an Authorization Code
```